### PR TITLE
Closes #79 - This adds Sticky Social Bar!

### LINKED ISSUE #79: Create new BB: Sticky Share Links
This is a Building Block for our upcoming News/Magazine Kit. For anyone who wants to contribute some building blocks to the site, feel free to assign this to yourself to let us know you're taking this one on!

**Description/Interaction:** A simple list of share links that will stay on the page as someone scrolls through a news article.

**Example:**
screen shot 2017-04-14 at 4 41 39 pm

### PR DESCRIPTION
**Pull request by IamManchanda**
{ No JS whatsoever, CSS only Sticky Social Bar }

Codepen Demo's: 
- Positioned Left => http://codepen.io/IamManchanda/full/QvbwvZ/
- Positioned Right => http://codepen.io/IamManchanda/full/ybNyor/

cc / @canterberrie @kball for review/merge
Poke @rafibomb for yml english :D

### CODE REVIEW

**Comment by IamManchanda** [comment]
I have no idea why github is showing so bad indentation for this ... i have indented properly as far of my best of knowledge

screenshot from 2017-04-16 01-44-45

File: src/building-blocks/sticky-social-bar/sticky-social-bar.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+$social-bar-position: left; // Change this value to `right` for changing sidebar's position.
+$social-bar-transformation: 140px;
+$social-bar-width: 180px;
```
Comment (canterberrie) [inline src/building-blocks/sticky-social-bar/sticky-social-bar.scss]: Can we use rem-calc here?

**Comment by kball** [comment]
@IamManchanda The indentation issue is probably that your editor is using tabs. Please configure to use spaces, with one tab equal to 2 spaces.

File: src/building-blocks/sticky-social-bar/sticky-social-bar.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet">
```
Comment (kball) [inline src/building-blocks/sticky-social-bar/sticky-social-bar.html]: We actually don't even need the link in the partial, as it is included in the layout by default

File: src/building-blocks/sticky-social-bar/sticky-social-bar.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: "Sticky Social Bar"
+tags:
+  - share
+  - social
+  - sticky
+  - position
+category: social
```
Comment (canterberrie) [inline src/building-blocks/sticky-social-bar/sticky-social-bar.yml]: "social" isn't a category (yet!). I think changing to "navigation" would be appropriate for this Building Block.

File: src/building-blocks/sticky-social-bar/sticky-social-bar.scss (hunk 1 above)
Comment (IamManchanda) [inline src/building-blocks/sticky-social-bar/sticky-social-bar.scss]: Yeah sure!

**Comment by IamManchanda** [comment]
done @kball @canterberrie

**Comment by kball** [comment]
Great! Merged!

**Pull request merged**